pages/people/[id]/warning-notes/[warningNoteId]/[[...stepId]].tsx

```diff
@@ -18,21 +19,17 @@ const ReviewWarningNote = (): React.ReactElement => {
   const confirmation = asPath.includes('confirmation');
   const { user } = useAuth() as { user: User };
   const onFormSubmit = useCallback(
-    (person: Resident) => async ({
-      form_name,
-      ...formData
-    }: Record<string, unknown>) => {
-      console.log({
-        personId: person.id,
-        firstName: person.firstName,
-        lastName: person.lastName,
-        contextFlag: person.contextFlag,
-        dateOfBirth: person.dateOfBirth,
-        workerEmail: user.email,
-        formNameOverall:
-          person.contextFlag === 'A' ? 'ASC_case_note' : 'CFS_case_note',
-        formName: form_name,
-        caseFormData: JSON.stringify(formData),
+    (person: Resident) => async ({ ...formData }: Record<string, unknown>) => {
```

Comment:
I have a feeling the spread inside a destructure isn't necessary here now you've removed the `form_name` argument. I think the below is equivalent (worth testing though!):

```suggestion
    (person: Resident) => async (formData: Record<string, unknown>) => {
```